### DISCUSSION TITLE
Github Actions deployment to Github Pages White Screen

### DISCUSSION THREAD

**Discussion by NathanScott85** [community/community#23683] [Other Feature Feedback, Questions, & Ideas]
<p>I have my own personal company website that I have been trying to set up a deployment to GitHub pages using GitHub Actions, I recently solved an issue with the organization deployment and I am now facing another issue of the website is stating it’s deployed but it’s just showing a White Screen.</p>
<p>I used a CRA(create-react-app) which you can view here:  -</p>
<p><a href="https://github.com/Saharadigital/sahara-digital" rel="noopener nofollow ugc">Gtihub Project Link</a></p>
<p><a href="https://github.com/Saharadigital/sahara-digital/actions" rel="noopener nofollow ugc">Github Actions Deployments</a>.</p>
<p>How to get this so that it does not display a white screen?</p>

**Comment by Maximization** [comment]
<p>Hi Nathan! If go to <a href="http://saharadigital.github.io/sahara-digital" rel="noopener nofollow ugc">saharadigital.github.io/sahara-digital</a> I am being redirected to <a href="https://saharadigital.co.uk/" rel="noopener nofollow ugc">https://saharadigital.co.uk/</a> which shows up just fine.</p>
<p>Did you fix the problem in the meanwhile?</p>

**Comment by NathanScott85** [comment, marked as answer]
<p>Hi,<br>
Yea I managed to fix the problem last night you can mark this as solved.</p>
<p>The solution to the issue was that I had some incorrect config in the package.json and also in the .yml file I did not need the username for the organization url.</p>
